core/src/main/resources/lib/form/password.jelly

```diff
@@ -61,12 +61,60 @@ THE SOFTWARE.
     </st:attribute>
   </st:documentation>
   <f:prepareDatabinding />
-  <m:input xmlns:m="jelly:hudson.util.jelly.MorphTagLibrary"
-         class="setting-input ${attrs.checkUrl!=null?'validated ':''}${attrs.clazz}"
-         name ="${attrs.name ?: '_.'+attrs.field}"
-         value="${h.getPasswordValue(attrs.value ?: instance[attrs.field])}"
-         type="password"
-         checkMethod="post"
-         ATTRIBUTES="${attrs}" EXCEPT="field clazz value" />
+  <j:set var="value" value="${h.getPasswordValue(attrs.value ?: instance[attrs.field])}"/>
+  <j:choose>
+    <j:when test="${ value != null }">
+      <st:adjunct includes="lib.form.password.password"/>
+      <div class="hidden-password">
+        <m:input xmlns:m="jelly:hudson.util.jelly.MorphTagLibrary"
+                 class="hidden-password-field setting-input ${attrs.checkUrl!=null?'validated ':''}${attrs.clazz}"
+                 name ="${attrs.name ?: '_.'+attrs.field}"
```

Comment:
```suggestion
                 name="${attrs.name ?: '_.'+attrs.field}"
```

Reply:
Preferring a nicer `?w=1`

Reply:
What?

Reply:
I thought this impacted the whitespace-ignoring diff but apparently it doesn't.

Reply:
Ohh, ok.